create-chroot.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e -u
+
+cd "$(dirname "$0")"
+
+if [ "${HOMEWORLD_CHROOT:-}" = "" -o ! -e "$(dirname HOMEWORLD_CHROOT)" ]
+then
+    echo "invalid path to chroot: ${HOMEWORLD_CHROOT:-}" 1>&2
+    echo '(have you populated $HOMEWORLD_CHROOT?)'
+    echo '(try export HOMEWORLD_CHROOT=$HOME/chroot)'
+    exit 1
+fi
+
+if [ -e "${HOMEWORLD_CHROOT}" ]
+then
+    echo "chroot already exists" 1>&2
+    exit 1
+fi
+
+mkdir "${HOMEWORLD_CHROOT}"
+sudo debootstrap --include="$(cat chroot-packages.list | grep -vE '^#' | tr '\n ' ',,' | sed 's/,$//')" stretch "${HOMEWORLD_CHROOT}" http://debian.csail.mit.edu/debian/
+sudo ln -sT "homeworld/building" "${HOMEWORLD_CHROOT}/h"
```

Comment:
Shouldn't the directory you're linking be building (instead of homeworld/building)? Also shouldn't all the chroot scripts be logically under building instead of the project root?

Reply:
I didn't want to have it directly under the project directory, because I didn't want a stray 'rm -rf' to erase a mountpoint on the real filesystem. This concern might be mitigated now that we're using nspawn-based isolation.